src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -116,14 +118,14 @@ public function __construct(ResourceMetadataFactoryInterface $resourceMetadataFa
         $this->paginationEnabled = $paginationEnabled;
         $this->paginationPageParameterName = $paginationPageParameterName;
         $this->apiKeys = $apiKeys;
-        $this->subresourceOperationFactory = $subresourceOperationFactory;
```

Comment:
Should be reverted I guess

Reply:
Actually this line is duplicate, check a few lines up. I just removed the duplicate line.